### PR TITLE
chore(*): Removes wascc from the repo

### PR DESCRIPTION
**Pull request by thomastaylor312**
Now that we have moved the wascc provider to its new home, we can remove
the dependency from here. We need to make sure we include an announcement
about this in the 0.7 release notes

### CODE REVIEW

File: docs/community/developers.md [hunk 1]
```diff
@@ -41,8 +41,8 @@ implementation):
 $ just build --no-default-features --features rustls-tls
 ```
 
-The same flags can be passed to `just run-wasi` and `just run-wascc` if you want
-to just [run](#running) the project instead.
+The same flags can be passed to `just run`. if you want to just [run](#running)
```
Comment (bacongobbler) [inline docs/community/developers.md:44]: looks like a period was added here by mistake?

File: docs/community/developers.md [hunk 2]
```diff
@@ -86,12 +86,7 @@ things to be aware of. See the [caveats](#caveats) section for more details
 
 ## Running
 
-There are two different runtimes available for Krustlet: `wascc` or `wasi`.
-
-The `wascc` runtime is a secure WebAssembly host runtime, connecting "actors"
-and "capability providers" together to connect your WebAssembly runtime to
-cloud-native services like message brokers, databases, or other external
-services normally unavailable to the WebAssembly runtime.
+There default included runtime with Krustlet is `wasi`.
```
Comment (bacongobbler) [inline docs/community/developers.md:89]: The*

**Review by bacongobbler — APPROVED** [review]
code LGTM

File: docs/community/developers.md (hunk 1 above)
Comment (thomastaylor312) [reply docs/community/developers.md:44]: Yes it was

File: docs/community/developers.md (hunk 2 above)
Comment (thomastaylor312) [reply docs/community/developers.md:89]: Good catch

**Pull request merged**